circle.yml

```diff
@@ -0,0 +1,45 @@
+# https://circleci.com/gh/spyder-ide/spyder
+
+machine:
+  environment:
+    # Used by qthelpers to close widgets after a defined time
+    TEST_CI: "True"
+    # Python versions to tests (Maximum of 4 different versions)
+    PY_VERSIONS: "2.7 3.4 3.5"
+    # Environment variables used by astropy helpers
+    TRAVIS_OS_NAME: "linux"
+    CONDA_CHANNELS: "spyder-ide"
+    CONDA_DEPENDENCIES: >
+      jedi mock nbconvert numpy pandas pep8 psutil pyflakes pygments pylint
+      pytest pytest-cov pytest-qt pyyaml qtawesome qtpy requests sphinx
```

Comment:
We don't need pyyaml and requests here.

Missing Spyder dependencies: qtconsole, rope, pyzmq and chardet.

Please also add nomkl as a dependency here to avoid installing mkl as part of numpy and pandas. Mkl is a huge package and so it slows down our tests.